src/commands/Register.ts

```diff
@@ -136,8 +136,9 @@ export class Register extends CommandClient {
 
 export async function executeRegister(registerOptions?: RegisterOptions) {
     const parameters: any = getParams();
-    const configFileName =
-        parameters.config ?? path.resolve('./server-config.json');
+    const configFileName = parameters.config
+        ? parameters.config
+        : path.resolve('./server-config.json');
```

Comment:
when will this not evaluate to `path.resolve('./server-config.json')`? how could it ever take the value of `parameters.config`?